modules/tinymce/changelog.txt

```diff
@@ -1,17 +1,17 @@
 Version 5.3.0 (TBD)
     Added html and body height styles to the default oxide content CSS #TINY-5978
-    Added `uploadUri` and `blobInfo` to `editorUpload.uploadImages` return type #TINY-4579
+    Added `uploadUri` and `blobInfo` to the data returned by `editor.uploadImages()` #TINY-4579
     Added new function to the `BlobCache` API to lookup a blob based on the base64 data and mime type #TINY-5988
     Added the ability to search and replace within a selection #TINY-4549
     Added the ability to set the list start position for ordered lists and added new `lists` context menu item #TINY-3915
     Added `icon` as an optional config option to the toggle menu item API #TINY-3345
     Changed toggle menu items and choice menu items to have a dedicated icon with the checkmark displayed on the far right side of the menu item #TINY-3345
-    Changed the `link`, `image` and `paste` plugins to use Promises to reduce the bundle size #TINY-4710
+    Changed the `link`, `image`, and `paste` plugins to use Promises to reduce the bundle size #TINY-4710
     Changed the default icons to be lazy loaded during initialization #TINY-4729
-    Changed so base64 encoded urls are converted to blob urls when content is parsed #TINY-4727
+    Changed base64 encoded urls are converted to blob urls when content is parsed #TINY-4727
```

Comment:
This also reads strangely now. Maybe it needs to be something like this instead?
```
Changed base64 encoded urls so they are converted to blob urls when the content is parsed #TINY-4727
```

Reply:
> Changed the parsing of content so base64 encoded urls are converted to blob urls

Reply:
Fixed by https://github.com/tinymce/tinymce/pull/5671/commits/3afd8db191c5d5fa0ad10ebcd1665364c10dcd66